src/blueprint/telemetry.rs

```diff
@@ -0,0 +1,72 @@
+use std::str::FromStr;
+
+use hyper::header::{HeaderName, HeaderValue};
+use hyper::HeaderMap;
+use url::Url;
+
+use super::TryFoldConfig;
+use crate::config::{self, ConfigModule, KeyValues, PrometheusExporter, StdoutExporter};
+use crate::directive::DirectiveCodec;
+use crate::try_fold::TryFold;
+use crate::valid::{Valid, ValidationError, Validator};
+
+#[derive(Debug, Clone)]
+pub struct OtlpExporter {
+    pub url: Url,
+    pub headers: HeaderMap,
+}
+
+#[derive(Debug, Clone)]
+pub enum TelemetryExporter {
+    Stdout(StdoutExporter),
+    Otlp(OtlpExporter),
+    Prometheus(PrometheusExporter),
+}
+
+#[derive(Debug, Clone)]
+pub struct TelemetryInner {
+    pub export: TelemetryExporter,
```

Comment:
Not required.

Reply:
removed that wrapper since it's not required